src/FunctionalGame.js

```diff
@@ -24,17 +29,13 @@ function FunctionalGame() {
     setHistory( actualHistory.concat([{ squares: squares }]));
     setStepNumber(actualHistory.length);
     setXIsNext(!xIsNext);
```

Comment:
This should be `setXIsNext(prev => !prev)` so you ensure you're updating toggling the correct value